cpu/atxmega/vendor/ebi/ebi.c

```diff
@@ -0,0 +1,375 @@
+/*
```

Comment:
The `vendor` directory is usually for code that was copied verbatim from the vendor, but this appears to be authored by you.

Why not just place this in `cpu/atxmega/ebi/` ?

Reply:
Yes, it can be. However, I would like avoid create a list of directories at cpu/atxmega.
Is is acceptable just rename vendor to drivers?

Reply:
I mean there is already precedent for just plopping in folders with driver code right into the folder of the respective CPU (usually ethernet drivers), they don't tend to become too many.

If you prefer a drivers folder that's fine too, just not `vendor` since that already has different semantics (and `vendor/` folders are exempted from code style check CI runs).

Reply:
Ahh ok, I'll follow convention.